release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3059,11 +3059,26 @@ The default `IngressController` CR is configured with 4 HAProxy threads. If you
 * For {sno-caps} on {product-version} and Google Cloud Platform (GCP), there is a known issue with the Cloud Network Config Controller (CNCC) entering a `CrashLoopBackOff` state. This occurs at initialization time when the CNCC tries to reach the GCP internal load balancer address and the resulting hairpin traffic is not correctly prevented in OVN-Kubernetes shared gateway mode on GCP causing it to get dropped. Cluster Network Operator will show a `Progressing=true` status in such case. Currently, there is no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGS-20554[*OCPBUGS-20554*])
 
 * On an {sno-caps} that has guaranteed CPUs and where Interrupt Request (IRQ) load balancing is disabled, large latency spikes can occur at container startup. (link:https://issues.redhat.com/browse/OCPBUGS-22901[*OCPBUGS-22901*])
++
+[NOTE]
+====
+This issue is fixed in 4.14.6.
```

Comment:
This looks good to me based on comments in [OCPBUGS-24281](https://issues.redhat.com/browse/OCPBUGS-24281)